backend/backend.config.js

```diff
@@ -57,5 +57,15 @@ module.exports = {
         parseInt(process.env.ACTIVE_ACCOUNTS_POLLING_TIME_MS, 10)
         || 6 * 60 * 60 * 1000, // 6 hours
     },
+    {
+      name: 'ddc',
+      enabled: !process.env.DDC_DISABLE,
+      crawler: './crawlers/ddc.js',
+      contractName: process.env.DDC_CONTRACT_NAME || 'ddc_bucket',
+      rpc: process.env.DDC_RPC || 'wss://rpc.v2.testnet.cere.network/ws',
```

Comment:
Does it make sense to use here `devnet` instead of the `testnet` subdomain?

Reply:
i think it's the same questions as above